app/views/organizations/_details.html.erb

```diff
@@ -146,6 +146,13 @@
                   <%= humanize_boolean(@organization.enable_quantity_based_requests) %>
                 </p>
               </div>
+              <div class="mb-4">
+                <h3 class='font-bold'>Show Year-to-date values on distribution printout?</h3>
+                <p>
+                  <%= fa_icon "calendar" %>
```

Comment:
I would remove the fa_icon "calendar" here,  because although the information relates to how we use dates, it is not a date.